### DISCUSSION TITLE
miising images on prompt message

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#70] [General]
When transaction such as a successful delegation, the following prompt message which is missing an image ?

![Screenshot 2023-05-24 at 19 33 17](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/5b3e1e5d-1c01-4216-9abe-fab209ee1626)

**Comment by TJ-BitCanna** [comment]
Thanks for notifying!

I'll close this one, since it was also recorded by @Lexey777 on this issue:
https://github.com/BitCannaGlobal/WeedWallet/discussions/66
